core/src/services/vercel_artifacts/backend.rs

```diff
@@ -95,15 +101,45 @@ impl Accessor for VercelArtifactsBackend {
             VercelArtifactsWriter::new(self.clone(), args, path.to_string()),
         ))
     }
+
+    async fn stat(&self, path: &str, _args: OpStat) -> Result<RpStat> {
+        if (path == "/") || (path.ends_with('/')) {
+            return Ok(RpStat::new(Metadata::new(EntryMode::DIR)));
+        }
+
+        let res = self.vercel_artifacts_stat(path).await?;
+
+        let status = res.status();
+
+        match status {
+            StatusCode::OK => {
+                let meta = parse_into_metadata(path, res.headers())?;
+                Ok(RpStat::new(meta))
+            }
+
+            _ => Err(parse_error(res).await?),
+        }
+    }
+
+    async fn create_dir(&self, _path: &str, _args: OpCreateDir) -> Result<RpCreateDir> {
```

Comment:
It seems odd to return `Ok` for unsupported operations.

Reply:
🤔 Removed. Will continue this work after refactoring our tests.